src/action.js

```diff
@@ -50,12 +50,12 @@ module.exports = function link(config, args) {
         registerDependencyIOS(dependencyConfig.ios, project.ios);
       }
 
-      if (project.android && dependencyConfig.assets) {
+      if (project.android && dependencyConfig.assets.length > 0) {
```

Comment:
mb it'll be more elegant to write `!emtpy(dependencyConfig.assets)`, but feel free not to do it, it's just a notice